contrib/bloop/src/python/pants/contrib/bloop/register.py

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
```

Comment:
We don’t use this anymore or `__future__`! And you can start using Py3 features like f strings here, as it seems like we won’t be cherry picking this into 1.17.x.